base/access_provider/sync_to_target/model.go

```diff
@@ -41,11 +41,26 @@ type Access struct {
 }
 
 type WhoItem struct {
-	Users          []string `yaml:"users" json:"users"`
-	Groups         []string `yaml:"groups" json:"groups"`
-	InheritFrom    []string `yaml:"inheritFrom" json:"inheritFrom"`
-	UsersInGroups  []string `yaml:"usersInGroups" json:"usersInGroups"`
-	UsersInherited []string `yaml:"usersInherited" json:"usersInherited"`
+	// Users contains all account names of users assigned to this access provider
+	Users []string `yaml:"users,omitempty" json:"users,omitempty"`
+
+	// Groups contains all account names of groups assigned to this access provider
+	Groups []string `yaml:"groups,omitempty" json:"groups,omitempty"`
+
+	// InheritFrom contains all access providers actual names in WHO part of this access provider
+	InheritFrom []string `yaml:"inheritFrom,omitempty" json:"inheritFrom,omitempty"`
+
+	// UsersInGroup contains all account names of users assigned to groups of this access provider
```

Comment:
Make more clear that it's only the groups linked to this AP directly (and not the inherited ones)